basicplot/test_data/data.tab

```diff
@@ -0,0 +1,8 @@
+genes	sample1	sample2	sample3
+gene1	1,5	10	26
```

Comment:
1,5 is not a number for R. This should be 1.5
But let's work with a better input data test. I'll send you a real one asap